.env

```
@@ -0,0 +1,3 @@
+MONGO_PASSWORD=1234
+MONGO_USER=Isa_tm_22
+MONGO_DATABASE=Qbit-Test
```

No exponer credenciales de la base.

Y en las .env poner credenciales locales

```suggestion
DB_URI="mongodb://localhost:27017/qbit"
```